daml_dit_ddit/subcommand_inspect.py

```diff
@@ -53,7 +52,6 @@ def subcommand_main(dit_filename: str):
 
     try:
         dabl_meta = accept_dabl_meta(contents[DABL_META_NAME])
-        check_experimental(dabl_meta)
```

Comment:
Are deprecated fields being checked when running the `inspect` subcommand? Should they?

Reply:
No. What happens in this PR is that `ddit build` moves all the data out of those fields. No real point to inspect.